### ISSUE TITLE
Entity is already tracked!

### ISSUE THREAD

**Issue by DestructDez1** [The-1-7-10-Pack/The-1.7.10-Pack#161]
Error Log: http://pastebin.com/U0h4NJVy

This started to happen today on the lasted version of the modpack. Immediately after starting the server before anyone logs in console is being spammed with that error. I've tried deleted the player data thinking the issue was with one of the players but that did not work. 

I would also like to point out that this started happening after updating to 0.7.0. I had to delete my original world after updating to 0.7.0 and now its happening to a fresh world.

The server was installed cleanly. As in no overwrites.
Java has been updated to the latest version
The server is hosted on my own server rack
 -Dual Intel Xeon 
 -92GB RAM (5GB Dedicated to MC)

Any advice short of starting a new world would be great.

**Comment by Flukethoughts** [comment]
don't start a new world this is how u update
keep the config folder and world folder
and then u delete everything else then upload everything to the server besides the config folder. Thats what I have done countless times.

**Comment by DestructDez1** [comment]
I always create a new folder for the new server then copy the world to that folder. This way I don't have to deal with mod issues.

**Comment by Flukethoughts** [comment]
here is the fastewst result
do you run the server on your desktop before hand?

**Comment by DestructDez1** [comment]
No i do not

**Comment by Flukethoughts** [comment]
hint hint hint

**Comment by DestructDez1** [comment]
So you are telling me to run a fresh server with my current world on my windows 8.1 PRO PC  then move the whole thing to my Server rack running windows server 2012? Mkay. Ill give it a shot and report back when happens.

**Comment by Flukethoughts** [comment]
trust me it's weird
I do it every time
I run it on ym desktop to confirm its working then move to the server in it's own enviroment. Through out all you know about servers things get weird with minecraft.

**Comment by DestructDez1** [comment]
Sorry. Same error spams the logs. The lag is so great that we time out before we can do anything. Sigh, I don't want to recreate a world again.

**Comment by DestructDez1** [comment]
http://prntscr.com/612k9a

In the screenshot you will see almost 34k mobs kills using the /xu_killmobs command. So it appears that mobs are being replicated. Could this be a specific mod issue?

**Comment by The-1-7-10-Pack** [comment]
You can try use Opis to locate the problem on your save.

**Comment by DestructDez1** [comment]
Cool beans! I'll give that a shot but what if the chunk causing an error doesnt have any player placed tiles. For example: The chunk was loaded naturally while exploring but was also glitched. I would think MCEdit could take care of the corrupted chunk but not the entities spawning on it. Regardless, thank you guys for the suggestions but if this doesn't work then I'm going to need to write a script to auto enter the kill entities command. They are still replicating at about 1k every 30 minutes.

**Comment by The-1-7-10-Pack** [comment]
Maybe it something on your save files, but I'm happy you have good ideas to fix your problem :)

**Comment by Flukethoughts** [comment]
u have all the mods dont you?

**Comment by da404lewzer** [comment]
I had this issue just recently, no resolution though :/ Are you also having issues with Wither Skeletons in the Nether?

https://github.com/The-1-7-10-Pack/The-1.7.10-Pack/issues/154

**Comment by DestructDez1** [comment]
The server and client are not changed from the original install.

I believe we found the issue. An ender pump placed close to a nether fortress was spawning Ender IO Wither skeletons continuously. Removing the ender pump fixed the error. I'll also report this to them when I find the right place to do so.

Once again, Thank you guys for the help!

**Comment by da404lewzer** [comment]
I too have an ender pump near my wither skeleton issue.. hrm

**Comment by DestructDez1** [comment]
Try moving it away from the fortress, That fixed the issue completely for me.

**Comment by The-1-7-10-Pack** [comment]
Good you fixed your issue! Be sure to report that bug there: https://github.com/rwtema/extrautilities/issues
